### DISCUSSION TITLE
進捗と設定値

### DISCUSSION THREAD

**Discussion by namikis** [jphacks/F_2109#28] [Show and tell]
@kenkenpa2126 
提出API～公開URL取得はほぼ完成しました。
あと設定が必要なのは、
１．問題の正解画像(width1200pxにしていただけるとありがたいです)
２．スクショの画像サイズ
３．公開URLのドメイン部分（EC2インスタンスのElasticIP）

です。

**Comment by nisshi-dev** [comment]
@namikis 
問題の正解画像についてなんですが、横幅375のものって取れますか？
取れそうだったらデフォルトそっち返す感じでお願いしたいです！！
